vividus-plugin-web-app/src/main/java/org/vividus/bdd/converter/StringToListSequenceActionConverter.java

```diff
@@ -51,8 +51,13 @@ public StringToListSequenceActionConverter(StringToSearchAttributesConverter str
             .map(params ->
             {
                 SequenceActionType actionType = params.valueAs("type", SequenceActionType.class);
+                String argumentAsString = argumentAs(params, String.class);
+                if (argumentAsString.isEmpty() && actionType.isNullable())
```

Comment:
and what about not nullable arguments? Shouldn't you notify the user about an invalid argument?